src/lib/ui/pressable/component.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Pressable, PressableProps, StyleSheet, StyleProp, ViewStyle } from 'react-native';
+
+export interface AppPressableProps extends PressableProps {
+  style?: StyleProp<ViewStyle>;
```

Comment:
Isn't `style` props presented in `PressableProps`? What's the purpose of `AppPressableProps`?